ApolloExtra.lua

```diff
@@ -3367,114 +3408,114 @@ OlympusLootData.ApolloUpgrade = {
 					PortraitExitWait = 1.0,
 					Emote = "PortraitEmoteAffection",
 					StartSound = "/Leftovers/World Sounds/MapZoomInShort", UseEventEndSound = true,
-					Text = "Lasting relationships are built on reciprocity. So if you lavish me with gifts, why, what am I to do but give you something in return?" },
+					Text = "I'd be lying if I said I didn't see that one coming. That's why I prepared this gift back. Now, we're even." },
 			},
 			ApolloGift02 =
 			{
                 Name = "ApolloGift02",
 				PlayOnce = true,
 				RequiredTextLines = { "ApolloGift01" },
-				{ Cue = "/VO/ZagreusHome_0536", Portrait = "Portrait_Zag_Default_01", Speaker = "CharProtag",
+				{ Cue = "/VO/ZagreusHome_3762", Portrait = "Portrait_Zag_Default_01", Speaker = "CharProtag",
 					PreLineAnim = "ZagreusTalkDenialStart", PreLineAnimTarget = "Hero",
 					PostLineAnim = "ZagreusTalkDenialReturnToIdle", PostLineAnimTarget = "Hero",
 					PostLineFunctionName = "BoonInteractPresentation",
-					Text = "My lord Apollo, if not for our acquaintance, I am certain that my days or nights would be much drearier indeed." },
-				{ Cue = "/VO/Apollo_0034",
+					Text = "Lord Apollo, you're light is very welcome in this darker part of the world. I must thank you for this." },
+				{ Cue = "/VO/Apollo_0123",
 					PortraitExitWait = 1.0,
 					Emote = "PortraitEmoteAffection",
 					StartSound = "/Leftovers/World Sounds/MapZoomInShort", UseEventEndSound = true,
-					Text = "Why, dearest, such a wonderfully thoughtful gift, for me? I love it, and I would be lying if I said you shouldn't have!" },
+					Text = "It must be hard to find such Nectar where you are. I truly apreciate the gesture and any future gift are unecessary." },
 			},
 			ApolloGift03 =
 			{
                 Name = "ApolloGift03",
 				PlayOnce = true,
 				RequiredTextLines = { "ApolloGift02" },
-				{ Cue = "/VO/ZagreusHome_0537", Portrait = "Portrait_Zag_Default_01", Speaker = "CharProtag",
+				{ Cue = "/VO/ZagreusHome_3763", Portrait = "Portrait_Zag_Default_01", Speaker = "CharProtag",
 					PreLineAnim = "ZagreusTalkDenialStart", PreLineAnimTarget = "Hero",
 					PostLineAnim = "ZagreusTalkDenialReturnToIdle", PostLineAnimTarget = "Hero",
 					PostLineFunctionName = "BoonInteractPresentation",
-					Text = "Lord Apollo, I'm afraid there really isn't much here worthy of you, but perhaps this can suffice." },
-				{ Cue = "/VO/Apollo_0035",
+					Text = "I know you said to stop this, Lord Apollo. This is just a little something for all your help." },
+				{ Cue = "/VO/Apollo_0124",
 					PortraitExitWait = 1.0,
 					Emote = "PortraitEmoteAffection",
 					StartSound = "/Leftovers/World Sounds/MapZoomInShort", UseEventEndSound = true,
-					Text = "My, but if this isn't such a treasure to behold, and I accept it as a sign of your affection! Of something even more, perhaps...?" },
+					Text = "Another gift? You know a great gift you could make me? Escape your eternal prison and come to Olympus." },
```

Comment:
"Another gift? You know what would be the best gift? Escaping your eternal prison and come to Olympus."